vue/pages/app/index.html

```diff
@@ -404,6 +404,19 @@ <h4><a href="/apps">APPS</a> > <span v-if="isNew()">{{ params.app.toUpperCase()
                               </div>
                             </div>
                         </div>
+                        <div v-if="app.type == 's3'" class="field is-horizontal">
+                          <div class="field-label is-normal">
+                            <label class="label">CloudFront (ID)</label>
```

Comment:
No need to put the `ID` in a parenthetical expression. The other fields are like that when they need to have distinguishing value since their names were the same.

Reply:
got it